server/service/client.go

```diff
@@ -421,7 +456,7 @@ func (c *Client) ApplyGroup(
 	if specs.AppConfig != nil {
 		windowsCustomSettings := extractAppCfgWindowsCustomSettings(specs.AppConfig)
 		macosCustomSettings := extractAppCfgMacOSCustomSettings(specs.AppConfig)
-		allCustomSettings := append(macosCustomSettings, windowsCustomSettings...)
+		allCustomSettings := append(macosCustomSettings, windowsCustomSettings...) // TODO: can we get rid of this?
```

Comment:
Is it used only for the condition on line  468 below? If so I guess it could be removed and the sum of the lengths be used instead.